app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionViewModel.kt

```diff
@@ -16,7 +17,10 @@ class AppVersionViewModel @Inject constructor(
   private val oppiaDateTimeFormatter: OppiaDateTimeFormatter
 ) : ObservableViewModel() {
 
-  val versionName = ObservableField<String>(BuildConfig.VERSION_NAME)
+  private lateinit var context: Context
+
+  var versionName = context.packageManager
```

Comment:
Prefer ``val``--we never want public state to be mutable.

Reply:
Got it.